doc/changes/changes_0.2.1.md

```diff
@@ -1,11 +1,19 @@
-# exasol-driver-lua 0.2.1, released 2023-06-??
+# exasol-driver-lua 0.2.1, released 2024-07-??
 
 Code name: Upgrade to Exasol 8
 
 ## Summary
 
-This release updates integration tests to additionally use Exasol version 8.18.1.
+This release updates integration tests to additionally use Exasol version 8.23.0.
+
+When running integration tests with `luarocks test` or `busted`, the environment variable `EXASOL_HOST` now has a default value `localhost` to make running tests more convenient.
```

Comment:
Just a side note regarding localhost.
https://www.youtube.com/watch?v=98SYTvNw1kw

Reply:
I would not use that default in production code but in a test scenario, it actually reduces potential failure points.